.github/workflows/master.yml

```diff
@@ -22,42 +22,4 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.FLYTE_BOT_PAT }}
           WITH_V: true
-          DEFAULT_BUMP: patch
-
-  goreleaser:
-    name: Release flytectl
-    runs-on: ubuntu-latest
-    needs: [ bump-version ]
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-        with:
-          fetch-depth: "0"
-      - name: Run GoReleaser dry run
-        uses: goreleaser/goreleaser-action@v2
-        with:
-          version: latest
-          args: release --rm-dist
-        env:
-          GITHUB_TOKEN: ${{ secrets.FLYTE_BOT_PAT }}
```

Comment:
Wont the removal of this now cause the automatic patch release to be stopped.
We should do the release similar to flyteidl where within the PR title itself.
```
- Use #patch, #minor or #major in the pull-request title to bump the corresponding version. Otherwise, the patch version
will be bumped. More details
```

Reply:
no both are independent bump version will only create a tag(patch release). The other release workflow triggers on tag event and run goreleaser.

In this case, release workflow will run for both case 
- If bump version workflow create a tag
- If We manually create a tag

Reply:
Clarified offline that the version tagged will be created for the minor version aswell in the title of the PR. Major release is currently manual.

And the release works if any version tag is present irrespective of manually created or through a github workflow.